tests/bson/bson-int64-ctor-002.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\BSON\Int64 constructor (64-bit int value)
+--SKIPIF--
+<?php if (8 !== PHP_INT_SIZE) { die('skip Only for 64-bit platform'); } ?>
+--FILE--
+<?php
+
+var_dump(new MongoDB\BSON\Int64(9223372036854775807));
+var_dump(new MongoDB\BSON\Int64(~9223372036854775807));
```

Comment:
```suggestion
/* Note: -9223372036854775808 is parsed as a double, since PHP applies the
 * negation operator after parsing the integer, which exceeds INT64_MAX.
 * Therefore, we can either negate INT64_MAX or use PHP_INT_MIN. */
var_dump(new MongoDB\BSON\Int64(~9223372036854775807));
```

I won't push for including this every time we use `~9223372036854775807` in a test, but having it documented at least once is a good idea. The ctor test seemed like the most recognizable place to do so.

Reply:
Changed this to use PHP_INT_MIN as we're on a 64-bit platform - that leaves no instances of negating `INT64_MAX`.